26/03/2021 01:32:02|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 01:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 01:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 01:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 01:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 01:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 01:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 01:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 04:00:12|INFO    |root                            |BAZARR All existing episode subtitles indexed from disk.|
26/03/2021 04:32:15|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 04:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 04:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 04:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 04:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 04:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 04:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 04:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 05:00:01|INFO    |root                            |BAZARR All existing movie subtitles indexed from disk.|
26/03/2021 05:43:20|INFO    |root                            |BAZARR Finished searching for Subtitles to upgrade. Check History for more information.|
26/03/2021 06:57:24|INFO    |root                            |BAZARR Running cache maintenance|
26/03/2021 07:32:05|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 07:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 07:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 07:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 07:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 07:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 07:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 07:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 10:32:05|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 10:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 10:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 10:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 10:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 10:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 10:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 10:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 13:32:04|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 13:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 13:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 13:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 13:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 13:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 13:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 13:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 16:32:06|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 16:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 16:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 16:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 16:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 16:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 16:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 16:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 17:43:19|INFO    |root                            |BAZARR Finished searching for Subtitles to upgrade. Check History for more information.|
26/03/2021 19:32:01|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/26 19:52, because of: ServiceUnavailable. Exception info: '503'|
26/03/2021 19:32:21|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 19:53:20|INFO    |root                            |Using opensubtitles again after 20 minutes, (disabled because: ServiceUnavailable)|
26/03/2021 19:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 19:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 19:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 19:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 19:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 19:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 19:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
26/03/2021 22:32:09|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
26/03/2021 22:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 22:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The_Avengers/The_Avengers_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 22:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 22:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Deadpool/Deadpool_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 22:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 22:53:20|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/Jojo_Rabbit/Jojo_Rabbit_BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
26/03/2021 22:53:20|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
